css/css-scroll-snap/scroll-target-padding-003.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-snap-padding with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#scroll-margin'>
+<link rel='match' href='scroll-target-001-ref.html'>
```

Comment:
An empty flags meta is a better way to say "no flags" than leaving it out, which might mean "no flags", or might mean "I forgot to consider flags". Not a show stopper, but I'd suggest you fix it.